### ISSUE TITLE
I have tried "python main.py", and installed TensorFlow via "pip install tessorflow", but it was reported like this: "ERROR: No matching distribution found for tensorflow" 

### ISSUE THREAD

**Issue by cheoljason** [kimbring2/DeepSoccer#8]
Error Message Following
~/catkin_ws/src/DeepSoccer/jetbot/jetbot_control/src$
$ python main.py
Traceback (most recent call last):
  File "main.py", line 16, in <module>
    import cvlib as cv
  File "/home/jetbot/.local/lib/python2.7/site-packages/cvlib/__init__.py", line 8, in <module>
    from .gender_detection import detect_gender
  File "/home/jetbot/.local/lib/python2.7/site-packages/cvlib/gender_detection.py", line 4, in <module>
    from keras.utils import get_file
  File "/home/jetbot/.local/lib/python2.7/site-packages/keras/__init__.py", line 6, in <module>
    'Keras requires TensorFlow 2.2 or higher. '
ImportError: Keras requires TensorFlow 2.2 or higher. Install TensorFlow via `pip install tensorflow`

**Comment by kimbring2** [comment]
@cheoljason 

Hello, cvlib requires a Tensorflow. You need to install it using 'pip install tensorflow'.

Thank you.

**Comment by cheoljason** [comment]
Thanks for answering. Though I have already i tried to install it using 'pip install tensorflow', it didn't work. What should I do to install tensorflow for Python2.7?

**Comment by kimbring2** [comment]
@cheoljason Oh, let's try 'pip2 install tensorflow'. You may need to check which python version is related with the pip. Sometimes, there are many environment option for Python.

**Comment by cheoljason** [comment]
I installed tensorflow today on the JetBot Python2.7 development environment.
I downloaded tensorflow_gpu-1.13.0rc0+nv19.2-cp27-cp27mu-linux_aarch64.whl from nvidia URL (https://developer.nvidia.com/embedded/downloads#?search=tensorflow) and barely installed tensorflow
The statement is as follows:
$ pip install tensorflow_gpu-1.13.0rc0+nv19.2-cp27-cp27mu-linux_aarch64.whl
$ pip install keras==2.3.1
$ pip install tensorflow-estimator==1.13.0
Nvidia does not currently provide python2.7 tensorflow, so it seems to have been installed complicatedly.

Thank you for opening and supporting kimbrings2's development materials for robot soccer.
